From 98b31e58c2ba76af58cbdea5fd78b34bd17e7770 Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Thu, 27 Oct 2022 10:40:50 +0200
Subject: [PATCH] Debugging

---
 setupinfo.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/setupinfo.py b/setupinfo.py
index 6758914..cc63fac 100644
--- a/setupinfo.py
+++ b/setupinfo.py
@@ -362,10 +362,11 @@ def run_command(cmd, *args):
     if args:
         cmd = ' '.join((cmd,) + args)
 
+    print("Running", cmd)
     p = subprocess.Popen(cmd, shell=True,
                          stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     stdout_data, errors = p.communicate()
-
+    print("stdout_data, errors =", stdout_data, errors)
     if p.returncode != 0 and errors:
         return ''
     return decode_input(stdout_data).strip()
@@ -405,6 +406,7 @@ def get_library_versions():
         return xml2_version, xslt_version
 
     # Path to xml2-config and xslt-config specified on the command line
+    print("{OPTION_WITH_XML2_CONFIG=}")
     if OPTION_WITH_XML2_CONFIG:
         xml2_version = get_library_version(OPTION_WITH_XML2_CONFIG)
         if xml2_version and OPTION_WITH_XSLT_CONFIG:
@@ -417,7 +419,9 @@ def get_library_versions():
     # Try pkg-config
     global PKG_CONFIG
     PKG_CONFIG = os.getenv('PKG_CONFIG', 'pkg-config')
+    print("{PKG_CONFIG=}")
     xml2_version = get_library_version(PKG_CONFIG, 'libxml-2.0')
+    print("{xml2_version=}")
     if xml2_version:
         xslt_version = get_library_version(PKG_CONFIG, 'libxslt')
         if xml2_version and xslt_version:
-- 
2.37.0 (Apple Git-136)

